clang/test/dpct/query_api_mapping/test.cu

```diff
@@ -17,3 +17,14 @@
 // CHECK-NEXT: Is migrated to (with the option --use-dpcpp-extensions=intel_device_math --use-experimental-features=bfloat16_math_functions):
 // CHECK-NEXT:   sycl::ext::intel::math::hfma(h1, h2);
 // CHECK-NEXT:   sycl::ext::oneapi::experimental::fma(b1, b2, b3);
+
+// RUN: not dpct --cuda-include-path="%cuda-path/include" --query-api-mapping=aaa 2>&1 | FileCheck %s -check-prefix=NO_MAPPING
+
+// NO_MAPPING: dpct exited with code: -43 (Error: This API mapping query is not available yet. You may get the API mapping by migrating a sample code of this API with the tool.)
+
+// RUN: not dpct --cuda-include-path="%cuda-path/include" --query-api-mapping=hdiv 2>&1 | FileCheck %s -check-prefix=PARSE_FAILED
+
+// PARSE_FAILED: {{(.+)}}temp.cu:7:3: error: use of undeclared identifier 'hdiv'
+// PARSE_FAILED-NEXT:     7 |   hdiv(h1 /*__half*/, h2 /*__half*/);
+// PARSE_FAILED-NEXT:       |   ^
+// PARSE_FAILED-NEXT: dpct exited with code: -17 (Error: Cannot parse input file(s))
```

Comment:
Please further update the L27~L30 error processing.  For end user, it doesn't make sense to see those line ,they want to know the hint why mapping not available:  1. API is not available in current CUDA header files, please use -cuda-sdk-header to provide 2.  API declare header file is not available.

Reply:
Please update it in separate PR.

Reply:
Will fix in another PR.